### ISSUE TITLE
[Bug] `Out-GridView` (PS7) color scheme has low contrast

### ISSUE THREAD

**Issue by GeeLaw** [PowerShell/PowerShell#12073]
# Steps to reproduce

```powershell
Get-Alias | Out-GridView -PassThru
```

Then select some items in the window.

# Expected behavior

The UI has sufficient contrast.

# Actual behavior

The contrast is insufficient. For example, odd-numbered rows (blue), hovered items (lighter blue), and selected items (darker blue) are blues that are very close.

<img width="462" alt="Screen Shot 2020-03-07 at 6 27 35 PM" src="https://user-images.githubusercontent.com/4462444/76155496-7d839d00-60a1-11ea-8121-64d4f256ce42.png">

In the screenshot above, item 2-4 are selected and item 8 is hovered.

# Environment data

```none
Name                           Value
----                           -----
PSVersion                      7.0.0
PSEdition                      Core
GitCommitId                    7.0.0
OS                             Darwin 19.2.0 Darwin Kernel Version 19.2.0: Sat …
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by iSazonov** [comment]
The code is frozen. https://github.com/PowerShell/GraphicalTools is for future.

**Comment by vexx32** [comment]
@iSazonov judging by the screenshot and OS information I'd say this was indeed the GraphicalTools module.

@GeeLaw could you open this issue on the GT repository that @iSazonov linked instead, please?

Thanks! 😊

**Comment by GeeLaw** [comment]
Thanks for the replies! I checked there and found PowerShell/GraphicalTools#33 fixed by PowerShell/GraphicalTools#35.
